### ISSUE TITLE
allowing higher filesizes in jsdos

### ISSUE THREAD

**Issue by JakeTrock** [caiiiycuk/js-dos.com#42]
I want to run openarena-dos on js-dos, how can I allow larger games to run in this environment

**Comment by caiiiycuk** [comment]
What game do you want to run? Please describe your problem.

**Comment by caiiiycuk** [comment]
Ok, sorry. Now I see game name, but do you have some error, or what?

**Comment by JakeTrock** [comment]
@caiiiycuk 
the command line says: 
![image](https://user-images.githubusercontent.com/17764452/45839229-c2ffaf00-bce1-11e8-928b-04bb4205c684.png)
For you to test it here is the directory I made, it runs on a nodejs server locally:
http://www.mediafire.com/file/1gxpivm628aomtx/qonline.zip/file

**Comment by JakeTrock** [comment]
@caiiiycuk do you think it was an error with how much ram dosbox provided? Although it runs on dos, Quake 3 probably uses a large amount of ram compared to dos games from the early 90's

**Comment by caiiiycuk** [comment]
Oh I am sorry, now I realized that file size is near 300Mb. I think it's not possible to use it in browser environment, because it breaks memory limitation of memory. Anyway even if you can run it, then you need to download 300 mb on start. It's too much. I am sorry.

**Comment by JakeTrock** [comment]
@caiiiycuk Does it matter that the zipped game folder is only actually 288mb?
![image](https://user-images.githubusercontent.com/17764452/46183863-e22dac00-c2a0-11e8-8ecb-9a8f3a8b3afb.png)

**Comment by caiiiycuk** [comment]
Anyway it's too large sorry

**Comment by JakeTrock** [comment]
ok. what is the right size?

**Comment by caiiiycuk** [comment]
Sorry, I was unclear. emscripten did not have limitation on size, I only mean that even if you run it, the game is not good for browser. Because 300Mb of download is too much. But issue it self is valid. Currently js-dos has limit 128Mb of ram, so it's not possible to extract this big archive. I think that limitation should be removed. But, I tried to run this game in native dos-box, and I saw that it's not dos game. I saw: ```This program can't run in DOS mode``` So it's not possible to run windows game inside dos-box/js-dos. Maybe you have DOS version of game, then I sure it possible to run it inside browser environment, just give me right archive and I do it. Thx.

CC @eyedeekay
